src/content/advanced/cluster-management/new-cluster-api/index.md

```diff
@@ -0,0 +1,124 @@
+---
+linkTitle: The new Cluster API
+title: The new Cluster API
+description: Benefits of the new Cluster API (based on CAPI)
+weight: 5
+menu:
+  main:
+    parent: advanced-cluster-management
+user_questions:
+  - What kind of changes are coming with the new Cluster API (CAPI)?
+  - What are the benefits of Cluster API (CAPI)?
+last_review_date: 2024-01-26
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-phoenix
+---
+
+{{< platform_support_table aws="ga=v21.0.0,ga=v21.0.0" azure="ga=v21.0.0" >}}
+
+## Embracing Kubernetes Cluster API
+### A Journey to a Unified Cluster Management Platform
+
+Two years ago, we embarked on a significant journey to transition our Kubernetes cluster management infrastructure from our own custom operators to the upstream Cluster API (CAPI) implementation. This decision was driven by a number of compelling factors, including the broader adoption and development of CAPI within the Kubernetes community, its ability to provide greater flexibility and configurability, and its commitment to future-proofing our platform.
+
+## Motivations for Transitioning to CAPI
+
+ * **Community-Driven Development**: CAPI is a widely adopted and actively developed project within the Kubernetes community. By embracing CAPI, we benefit from the collective expertise of many contributors, ensuring the continued advancement and reliability of our cluster management solution.
+ * **Vendor-Neutrality and Future-Proofing**: CAPI adheres to a neutral and open API, shielding our customers from vendor lock-in and ensuring their ability to seamlessly integrate with the evolving cloud landscape. This open approach also facilitates faster innovation and integration with new technologies.
+ * **Greater Flexibility and Customization**: CAPI provides a more expressive and flexible API for defining and managing Kubernetes clusters. With CAPI, we can cater to diverse customer requirements, enabling them to deploy into existing infrastructure and customize cluster configurations to suit their specific needs.
+ * **Expanded Tooling Ecosystem**: CAPI is compatible with a vast array of tools and solutions developed by the community. By adopting CAPI, we gain access to a rich toolkit for cluster management, automation, and orchestration.
+
+## Benefits of CAPI
```

Comment:
I see the benefits are quite similar to the motivations above. If we want to describe separately what motivations drive us as a company to do the transition to CAPI and what are the benefits to our customers as opposed we could try to be more specific. I will try to suggest something